docs/src/ci-intro.md

```diff
@@ -25,22 +23,18 @@ When [installing Playwright](./intro.md) using the [VS Code extension](./getting
 
 Playwright tests can be ran on any CI provider. In this section we will cover running tests on GitHub using GitHub actions. If you would like to see how to configure other CI providers check out our detailed doc on Continuous Integration.
 
-To add a [GitHub Actions](https://docs.github.com/en/actions) file first create `.github/workflows` folder and inside it add a `playwright.yml` file containing the example code below so that your tests will run on each push and pull request for the main/master branch.
-
 #### You will learn
 * langs: python, java, csharp
```

Comment:
just double check this makes sense for all langs except js or it should also have js?

Reply:
Yes this is intended. There's a separate "You will learn" list for JS that has different contents in https://github.com/microsoft/playwright/pull/31988/files#diff-49b43d48c93c3dc274299a0604e3a7c8ea1ed9891a230f3125e9992867d2ed3fL13-L14.